components/dashboard/src/teams/TeamBilling.tsx

```diff
@@ -34,6 +34,17 @@ export default function TeamBilling() {
     const [pendingTeamPlan, setPendingTeamPlan] = useState<PendingPlan | undefined>();
     const [pollTeamSubscriptionTimeout, setPollTeamSubscriptionTimeout] = useState<NodeJS.Timeout | undefined>();
 
+    console.log(
+        "members",
+        members.length,
+        "currency",
+        currency,
+        "teamSubscription",
+        teamSubscription,
+        "pendingTeamPlan",
+        pendingTeamPlan,
+    );
+
```

Comment:
**question**: Last comment on irrelevant line of code: Instead of the hover state as used with the editor and theme in preferences, we could use here a clear action button for each case, but we'd need to drop the hover state on the card. WDYT?

| BEFORE | AFTER |
|-|-|
| <img width="535" alt="Frame 1088" src="https://user-images.githubusercontent.com/120486/169902380-e978276e-a795-4bb5-8a0b-3fe33e1332e9.png"> | <img width="535" alt="Frame 1089" src="https://user-images.githubusercontent.com/120486/169902393-926c9bd4-0738-4d85-a6b4-18de0054698d.png"> |